week3/about_me_app/app/controllers/about_page_controller.rb

```diff
@@ -0,0 +1,24 @@
+class AboutPageController < ApplicationController
+    def home
+    end
+    def page
+        @name = params[:name]
+        @email = params[:email]
+    end
+    def form
+    end
+    def json
+        respond_to do |format|
+            format.html {redirect_to root_path} 
+            format.any {render plain: "Invalid Request Format", status: :bad_request}
+            format.json {
+                if params[:message]!=nil
+                    render json: {"message": params[:message]}, status: :ok
+                else
+                    render json: {"message": "INVALID",status: :bad_request}
```

Comment:
The `status` should be outside the `json` block
```suggestion
                    render json: { message: "INVALID" }, status: :bad_request
```

Reply:
I have made the changes and the code works now. Please review it.